### ISSUE TITLE
Request to upgrade analytics-node to resolve security issue in Axios

### ISSUE THREAD

**Issue by algonzalez-wf** [DavidWells/analytics#412]
Hello,

Snyk analysis shows a CSRF vulnerability in the axios version pulled in by @analytics/segment@1.1.4 › analytics-node@3.5.0 › axios@0.21.4. It also states the vulnerability is fixed in axios 1.6.0

Description: https://security.snyk.io/vuln/SNYK-JS-AXIOS-6032459

Not sure how much effort since latest release of analytics-node is 6.2.0 and axios is up to 1.6.2.
Also, it looks like analytics node has axios pinned at "^0.27.2"

Thank you and let me know if you need additional information.

**Comment by DavidWells** [comment]
Released updated analytics-node package via `@analytics/segment@2.0.0`

This is breaking change as `analytics-node` has changed since previous versions. New settings listed here: https://github.com/segmentio/analytics-next/blob/72dc35a46d7ba8ab90cab7a5190e9820c7dd0765/packages/node/src/app/settings.ts#L4C36-L43C2

**Comment by lleger** [comment]
@DavidWells This doesn't resolve the vulnerability as far as I can tell because `@analytics/segment@2.0.0` still depends on `analytics-node ^6.2.0` which has a vulnerable axios version pinned. But it's deprecated, so it looks like `@analytics/segment` needs to upgrade to the successor `@segment/analytics-node`.

See below for `yarn.lock`:

```
"@analytics/segment@2.0.0":
  version "2.0.0"
  resolved "https://registry.yarnpkg.com/@analytics/segment/-/segment-2.0.0.tgz#185bf2d073fd8d6b89943b0efd7c5e6ddf7e3cda"
  integrity sha512-Od86K85MNe/IOKnlAx26fl7mKG9hkPUn/Xx8lpqztRuoaR9+DgVJV3nKeDJJ6bbu6SvpSF0qmfsnQw0F0uisqg==
  dependencies:
    analytics-node "^6.2.0"
```

See below for `yarn audit`:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ moderate      │ Axios Cross-Site Request Forgery Vulnerability               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ axios                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=1.6.0                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @analytics/segment                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @analytics/segment > analytics-node > axios                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1094889                     │
└───────────────┴──────────────────────────────────────────────────────────────┘
```
